cli/Makefile

```diff
@@ -0,0 +1,25 @@
+TMP_DIR :=$(shell mktemp -d)
+
+all: codegen
+
+build: all
+	go build
+
+clone: 
+	@echo "Cloning eigenlayer-contracts..."
+	git clone -b feat/partial-withdrawal-batching --depth=1 git@github.com:Layr-Labs/eigenlayer-contracts.git $(TMP_DIR)
+	
+abi: clone
+	@echo "Generating abi..."
+	cd $(TMP_DIR) && make bindings
+	./fixAbi.sh $(TMP_DIR)/out/EigenPod.sol/EigenPod.json
+
+codegen: abi
+	@echo "Generating golang..."
+	abigen --abi $(TMP_DIR)/out/EigenPod.sol/EigenPod.json --pkg onchain --type EigenPod --out onchain/EigenPod.go
```

Comment:
Any reason we're not just importing the already generated binding from [eigenlayer-contracts](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/pkg/bindings/EigenPod/binding.go)?

Reply:
i tried importing these directly and deleting my abigen code, but I was getting dependency issues with pulling in the other repo entirely

happy to sit down monday and remove this code / figure that out, just didn't prioritize debugging it :)